### PR TITLE
Validation implemented

### PR DESCRIPTION
**Pull request by BilalLiaquat7**
In this PR: I have done with validation of the columns in rails

### CODE REVIEW

File: db/schema.rb
Comment (aamirkhan2478) [inline db/schema.rb:1]: https://github.com/BilalLiaquat7/blog_app/blob/08b77d8e1d18bd338ba97c09ee480006c1d15ac3/config/database.yml#L7-L10

[OPTIONAL BUT RECOMMANDED] Please use Microverse's suggested database `PostgreSQL`  but you using `Sqlite3` which is not a required database so kindly don't skip this optional requirement :slightly_smiling_face:

File: test/models/like_test.rb
Comment (aamirkhan2478) [inline test/models/like_test.rb:1]: - [ ] Please use the `Rspec` gem for testing but for now, you didn't use `Rspec` so kindly use the `Rspec` gem for testing for up your Rspec gem kindly follow the [Rspec documentation](git@github.com:rspec/rspec-rails.git) and write all test for all models and validations :slightly_smiling_face:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @BilalLiaquat7, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Bilal,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on implementing recommended changes!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
Let's make the ReadMe stand out even better by including instructions under the _Getting Started_ section, on how to run test cases for the app, for instance, using the command  `rspec`.

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**